### DISCUSSION TITLE
Does the Endpoint example really work?

### DISCUSSION THREAD

**Discussion by cshimegi** [finagle/finch#1662] [Q&A]
Hi, 
### What I'm trying
I'm new to Scala and I'm trying to follow [the example](https://github.com/finagle/finch/blob/v0.32.0/core/src/main/scala/io/finch/Endpoint.scala#L34C1-L42) but I'm facing some issues.

### Versions
Scala: 2.13.10
Dependencies:
```
libraryDependencies ++= Seq(
  "com.github.finagle" %% "finch-core" % 0.34.0,
  "com.github.finagle" %% "finch-circe" % 0.34.0,
  "io.circe" %% "circe-generic" % 0.14.3
)
```

### Development Env
1. IDE: IntelliJ
2. Mac Intel Chip

### Codes
```scala
import com.twitter.finagle.http.{ Request, Response }
import io.finch._
import com.twitter.finagle.{ Http, Service }
import com.twitter.util.Await

object Main extends App {
  case class Foo(i: Int)
  case class Bar(s: String)
  
  val foo: Endpoint[Foo] = get("foo") { Ok(Foo(42)) }
  val bar: Endpoint[Bar] = get("bar" :: path[String]) { s: String => Ok(Bar(s)) }

  private val api: Service[Request, Response] =
    (foo :+: bar).toService

  Await.ready(Http.server.serve(":8081", api))
}
```

### Issues
1. An error says "Cannot resolve symbol get" when using get method. Is this expected?
2. It seems "io.finch.syntax._" is gone after 0.32.0 version. When using get method with this version, it will require me to import syntax package. Just wondering is there an alternative way to do the similar thing?
3. How can I solve these errors? Seems no suggestions or good information on the Internet.

<img width="894" alt="Screenshot 2024-01-06 at 00 43 16" src="https://github.com/finagle/finch/assets/67813918/28e62e63-c283-48a5-b831-0387e8ff5190">

<img width="536" alt="Screenshot 2024-01-06 at 00 43 44" src="https://github.com/finagle/finch/assets/67813918/826d08aa-4a6e-49a2-b6ce-116340ec3fe9">

<img width="636" alt="Screenshot 2024-01-06 at 00 44 12" src="https://github.com/finagle/finch/assets/67813918/714af31f-b785-4109-b1bf-4d4aacb0b477">

Thank you for your help in advance!

**Comment by joroKr21** [comment]
It looks like the documentation for the website hasn't been updated. You can find a working example in the README for now.

**Reply by cshimegi** [reply]
Hi @joroKr21 
Thank you for your response. I'll try that!

**Comment by cshimegi** [comment]
Hi @joroKr21 
Sorry for another question. Do you know how to return Json response?
From examples (not only in the Readme but also in [this one](https://github.com/finagle/finch/tree/master/examples)), the response I can see is text and it will be transformed to JSON at the frontend side. But I don't like this way because it makes me feel not intuitive.

**Reply by cshimegi** [reply]
Ah, never mind. I found out how to do it.
